seed.js

```diff
@@ -108,15 +144,16 @@ const seed = async () => {
         senderId: userIds[0],
       },
     ];
+
     for (let i = 0; i < GENERATE_MESSAGES; i++) {
       messageList.push({
         fileType: postHelper(i),
         messageTitle: faker.hacker.phrase(),
         messageContent: faker.lorem.paragraph(),
         positiveVotes: Math.floor(Math.random() * Math.floor(600)),
         negativeVotes: Math.floor(Math.random() * Math.floor(600)),
-        latitude: (Math.random() * 100).toFixed(4),
-        longitude: (Math.random() * -100).toFixed(4),
+        latitude: generateRandomLat(),
+        longitude: generateRandomLong(),
         radius: Math.random() * (4 - 1) + 4,
         encrypted: faker.random.boolean(),
         key: faker.internet.password(),
```

Comment:
```suggestion
        channelId: channelIds[i + 1],
        senderId: userIds[i + 1],
```

This is going to error because `channelIds` now only has a length of 4. Might want to change it to choose a random `channelId` between index 0 & 4